content/docs/APIReference-LineContext.md

```diff
@@ -480,6 +482,66 @@ context.replyImageCarouselTemplate('this is an image carousel template', [
 
 <br />
 
+### Reply Flex Messages
+
+## `replyFlex(altText, contents)` - [Official Docs](https://developers.line.me/en/docs/messaging-api/reference/#flex-message)
```

Comment:
@XcrossD Thanks for sending this PR.
We use H4 markdown format (`####`) for method API.
Could you modify them to fit the convention?